tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2277,6 +2277,13 @@ voq/test_fabric_reach.py:
       - "not any(i in platform for i in ['arista_7800', 'x86_64-nokia_ixr7250e'])"
       - "(asic_type in ['cisco-8000'])"
 
+voq/test_voq_counter.py::test_voq_drop_counter:
```

Comment:
I have updated the issue with information on how to trigger this. Please try and let me know.